docs/0.5/howto/removing_a_splinter_circuit.md

```diff
@@ -0,0 +1,366 @@
+# Removing a Splinter Circuit
+
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+This guide explains alternative approaches to delete a circuit, disbanding and
+abandoning. Both of these processes result in removing the circuit’s
+networking, allowing for the circuit to be purged in either case. Disbanding a
+circuit begins with a node’s administrator proposing to remove a circuit’s
+networking functionality. Then, all members are able to vote before the circuit
+is disbanded. Once the circuit is disbanded, the circuit will no longer be
+usable to send messages and the circuit is only available to each node locally.
+On the other hand, a node’s administrator is able to abandon a circuit,
+breaking the circuit’s networking ability for all other circuit members. Both
```

Comment:
Break this into a new paragraph starting with "Both". Alternatively, this paragraph should be shortened, to be a simple introduction (it includes a little too much detail e.g. the voting stuff seems to detailed).

Reply:
I took out a lot of the specific details from this beginning paragraph, moved those details further down